packages/react-server-test-pages/package.json

```diff
@@ -19,5 +20,8 @@
     "react-server-cli": "^0.2.6",
     "react-server-data-bundle-cache": "^0.2.6",
     "superagent": "1.2.0"
+  },
+  "devDependencies": {
+    "node-inspector": ""
```

Comment:
Generally the easiest way to add a dependency like this is to use:

```bash
npm install --save-dev node-inspector
```

That will install it and add it to the `devDependencies` in package.json for you, with an appropriate version.